config/dev.sample.json

```diff
@@ -1,6 +1,6 @@
 {
-   "CONTRACT_ADDRESS": "0x5a76b2e98ae411e4791b9dd8f234947ade05ecf0",
```

Comment:
this change doesn't matter just that it is not necessary. this is just a sample file. you probably updated this by mistake.

Reply:
Submitted by mistake. I'll revert it